sycl/test-e2e/Matrix/element_wise_irreg_sum_rows_impl.hpp

```diff
@@ -44,8 +44,7 @@ void matrix_sum_rows(queue q, big_matrix<T, M, N> &B, nd_range<2> &r) {
 
            sycl::sub_group sg = spmd_item.get_sub_group();
 
-           joint_matrix<sub_group, T, use::b, TK, TN,
-                        ext::intel::experimental::matrix::layout::packed>
+           joint_matrix<sub_group, T, use::b, TK, TN, layout::ext_intel_packed>
```

Comment:
if element_wise_irreg_sum_rows.cpp was removed, I think this file also should be removed.

Reply:
Also, corresponding test for XMX8 should be removed as well. What about Legacy folder, would it be reasonable to remove it from Legacy as well for consistency and to avoid any potential future maintenance?